### ISSUE TITLE
does not work with webpack 4

### ISSUE THREAD

**Issue by francoisromain** [jamesjieye/html-webpack-exclude-assets-plugin#11]
Hello and thank you for the useful plugin. 
Today, Webpack updated to v.4 (with [webpack-contrib/html-webpack-plugin](https://github.com/webpack-contrib/html-webpack-plugin)) and this plugin now makes an error: 

```
ERROR in   TypeError: callback is not a function

  - index.js:30   [html-webpack-exclude-assets-plugin]/index.js:30:7
  - new Promise
  - Hook.js:35 AsyncSeriesWaterfallHook.lazyCompileHook [as _promise]
    [html-webpack-plugin]/[tapable]/lib/Hook.js:35:21
  - index.js:679 [html-webpack-plugin]/index.js:679:45
  - index.js:170  [html-webpack-plugin]/index.js:170:16
  - util.js:16 tryCatcher  [bluebird]/js/release/util.js:16:23
  - promise.js:512 Promise._settlePromiseFromHandler   [bluebird]/js/release/promise.js:512:31
  - promise.js:569 Promise._settlePromise   [bluebird]/js/release/promise.js:569:18
  - promise.js:614 Promise._settlePromise0   [bluebird]/js/release/promise.js:614:10
  - promise.js:693 Promise._settlePromises   [bluebird]/js/release/promise.js:693:18
  - async.js:133 Async._drainQueue  [bluebird]/js/release/async.js:133:16
  - async.js:143 Async._drainQueues  [bluebird]/js/release/async.js:143:10
  - async.js:17 Immediate.Async.drainQueues   [bluebird]/js/release/async.js:17:14

```

**Comment by jamesjieye** [comment]
[html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin/issues/860) has caused build failure. It hasn't upgraded to support webpack 4.0. 

Will work on this once html-webpack-plugin is ready for 4.0.

Thanks!

**Comment by francoisromain** [comment]
[webpack-contrib/html-webpack-plugin](https://github.com/webpack-contrib/html-webpack-plugin) is updated for webpack 4 already, and causing the bug reported above

**Comment by jamesjieye** [comment]
Fixed in version v0.0.6

**Comment by francoisromain** [comment]
thank you!
